plugin/loop/setup.go

```diff
@@ -73,8 +73,8 @@ func parse(c *caddy.Controller) (*Loop, error) {
 			return nil, c.ArgErr()
 		}
 
-		if len(c.ServerBlockKeys) > 0 {
-			zone = plugin.Host(c.ServerBlockKeys[0]).Normalize()
+		if len(c.Key) > 0 {
```

Comment:
I don't understand this change

Reply:
if you have a server bloc with that header:

domain1, domain2 {
   loop
}

the structure loop will be created twice, one for each Key of the bloc.

the setup of loop will be called 2 times

one with the following informations:

```
c.ServerBlockKeys = ["domain1", "domain2"]
c.Key = "domain1"
```
and second time with:
```
c.ServerBlockKeys = ["domain1", "domain2"]
c.Key = "domain2"
```

if you keep creating the Loop with c.ServerBlockKeys[0], you will always send a query to the domain "domain1" even though you are serving "domain2".